### ISSUE TITLE
Specific Categories for Photos?

### ISSUE THREAD

**Issue by sungkhum** [inspireui/beonews#43]
Is there a way to choose specific categories for the "Photo" section of the app?
Also, instead of grabbing the featured image of those posts, would it be possible to grab the Open Graph image instead (og:image)?

**Comment by inspireui** [comment]
Yes, this is the feature that we are checking solution to make new UI design as it's quite simple at the moment - thank you for your suggestion!
